beacon-chain/sync/rpc_ping.go

```diff
@@ -93,6 +93,9 @@ func (r *Service) validateSequenceNum(seq uint64, id peer.ID) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+	if md == nil {
+		return false, nil
```

Comment:
this should actually return true then, we will need to request the peer's metadata in that case